src/open-id-connect-authorize-step.ts

```diff
@@ -49,4 +54,9 @@ export default class OpenIdConnectAuthorizeStep implements PipelineStep {
       instruction.config.settings.roles !== undefined &&
       instruction.config.settings.roles.includes(role));
   }
+
+  private getInstructionUrl(instruction: NavigationInstruction): string {
```

Comment:
Off the top of my head, I don't think we do. I like the idea, though, of having a single responsibility service, thought, that provides additional information about a `NavigationInstruction`. It could have methods that receive a `NavigationInstruction` and return specific projections. So maybe it's a converter. 
    
    class NavigationInstructionConverter 
    {
        public ToAureliaRoute(NavigationInstruction instruction) { }
    }